src/pages/components/dropdown/accessibility.mdx

```diff
@@ -6,98 +6,129 @@ description:
+  option with focus. Pressing Esc cancels and closes the interaction.
+</Caption>
+
+</Column>
+</Row>
+
+#### Multiselect
+
+The multiselect opens with the `Enter` or `Space` keys, which are also used to
+toggle the selection of any option. Once items are selected, a tag appears in
+the field, showing a numerical representation of the number of items selected
+along with an ‘x’. Pressing `Esc` closes the multiselect. Pressing `Delete`
+while focus is in the collapsed field will clear all selected options.
 
 <Row>
-  <Column noGutterSm>
-    <StructuredListWrapper>
-      <StructuredListHead>
-        <StructuredListRow head>
-          <StructuredListCell head>
-            Automated test environment
-          </StructuredListCell>
-          <StructuredListCell head>Results</StructuredListCell>
-        </StructuredListRow>
-      </StructuredListHead>
-      <StructuredListBody>
-        <StructuredListRow>
-          <StructuredListCell>
-            - macOS Mojave version 10.14.2 with VoiceOver
-            <br />
-            - Chrome version 77.0.3865.90
-            <br />
-            - Dynamic Assessment Plugin (DAP) version 1.8.0.0 - IBM
-            Accessibility WCAG 2.1 Sept. 2019 Ruleset
-            <br />- Carbon React version 7.7.1
-          </StructuredListCell>
-          <StructuredListCell>
-            <strong>DAP test:</strong>
-            <br />- Violations
-          </StructuredListCell>
-        </StructuredListRow>
-      </StructuredListBody>
-    </StructuredListWrapper>
-  </Column>
+<Column colLg={8}>
+
+![Space and Enter keys open multiselects and toggle the selection of checkbox options. Esc closes.](images/dropdown-accessibility-3.png)
+
+<Caption>
+  Space and Enter keys open multiselects and toggle the state of the checkboxes
+  in the list. Esc closes the list.
+</Caption>
+
+</Column>
 </Row>
+
+#### Combo box
+
+The combo box has significantly different operation, since it is a combination
+of a text input and a dropdown. `Enter` or the `Up` or `Down` arrow keys will
+open the list of options, as will typing any character or string of characters
+(which will also filter the list to only show items that match the typed
+string). Pressing `Esc` will clear the input and collapse the list. Pressing
+`Enter` will select a highlighted option and collapse the list. (If nothing is
+highlighted in the list, pressing `Enter` will just toggle the combo box open
+and closed.) `Space` cannot be used for selecting, as pressing it will submit a
+space character into the filter string.
+
+<Row>
+<Column colLg={8}>
+
+![The Enter key opens and selects items in a combo box. Arrow keys can open and navigate the options. Esc clears the input and closes.](images/dropdown-accessibility-4.png)
+
+<Caption>
+  Combo boxes can be opened with the Enter or Up and Down arrow keys. Enter
+  selects the highlighted option in the list. Esc clears the input and closes
```

Comment:
The same thing with adding "Pressing Esc..." would be consistent with the caption before this.

Reply:
LOL, okay, so I just made the consistency the other way :) 
I don't think anyone can misread, and whatever can make captions a bit shorter, I tend to be keen to do.